CHANGELOG.md

```diff
@@ -21,6 +25,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 
 
-[Unreleased]: https://github.com/giantswarm/gscliauth/compare/v0.1.0...HEAD
+[Unreleased]: https://github.com/giantswarm/gscliauth/compare/v0.2.0...HEAD
+
+[0.2.0]: https://github.com/giantswarm/gscliauth/compare/v0.1.0...v0.2.0
 
```

Comment:
```suggestion

```

Reply:
I know you guys keep suggesting this but I do it on purpose.